person.rb

```diff
@@ -1,8 +1,10 @@
-class Person
+require_relative 'nameable'
```

Comment:
- [ ] Kindly ensure the name of the file matches the file importing. `nameable` and `name_able` are two different files